drivers/fastboot/fb_fsl/fb_fsl_partitions.c

```diff
@@ -46,14 +46,27 @@ enum {
 #endif
 	PTN_ALL_INDEX,
 	PTN_BOOTLOADER_INDEX,
+#ifdef CONFIG_FSL_FASTBOOT_BOOTLOADER_SECONDARY
+	PTN_BOOTLOADER_SECONDARY_INDEX,
+#endif
 #ifdef CONFIG_FSL_FASTBOOT_BOOTLOADER2
 	PTN_BOOTLOADER2_INDEX,
+	PTN_SIT_INDEX,
+#ifdef CONFIG_FSL_FASTBOOT_BOOTLOADER_SECONDARY
+	PTN_BOOTLOADER2_SECONDARY_INDEX,
+#endif
 #endif
 };
 
 struct fastboot_ptentry g_ptable[MAX_PTN];
 unsigned int g_pcount;
 
+static ulong secondary_image_table_mmc_offset(void)
```

Comment:
Might cause a defined but not used warning when CONFIG_FSL_FASTBOOT_BOOTLOADER_SECONDARY is not defined.